lib/matplotlib/axes/_axes.py

```diff
@@ -7,12 +7,12 @@
 import numpy as np
 from numpy import ma
 
-import matplotlib.category as _  # <-registers a category unit converter
```

Comment:
Actually, I like the idea of `import as _` to mark an unused import, but won't insistent on it.

Reply:
It is a nice way to mark that something a bit funny (side effects on import) is going on here.

Reply:
I rewrote the comment to more forcefully point to the side-effect, but I can also just revert the change and flake8-exclude this file if you prefer.

Reply:
My preferred version would be,

~~~
import matplotlib.category as _  # Register category unit converter as side-effect
~~~

Reply:
Ah, I guess I forgot to mention that this is needed because the underscore-import triggers F811 per https://github.com/PyCQA/pyflakes/issues/366.
I just removed the exclamation marks for now.